tests/suites/0_stateless/06_show/06_0001_show_create_table.sql

```diff
@@ -1,11 +1,11 @@
 DROP DATABASE IF EXISTS `test`;
 CREATE DATABASE `test`;
 CREATE TABLE `test`.`a` (
-    a bigint, b int default 3, c varchar(255) default 'x', d smallint null, e Date
+    a bigint, b int default 3, c varchar default 'x', d smallint null, e Date
```

Comment:
Should we accept varchar(...)`? Since we don't need the length anyway.

Reply:
Currently, it does throw an exception like this:

```
---------- Input ----------
create table a (c varchar(10))
---------- Output ---------
error: 
  --> SQL:1:26
  |
1 | create table a (c varchar(10))
  | ------                   ^ expected token ")"
  | |                         
  | while parsing `CREATE TABLE [IF NOT EXISTS] [<database>.]<table> [<source>] [ENGINE = <engine>]`
```

Reply:
Do we have plan to support  PG/MySQL dialect? Should not set standards(any of PG/MySQL), adapt them through dialect :D
For MySQL users, `varchar(N)` is very common.

Reply:
`varchar(N)` is common, but we don't have a plan to support this datatype.  So better throw errors to tell users that we don't support zero-filled fixed varchar datatype.

Reply:
Seems we can map `x varchar(N)`  to `x varchar  check(length(x) <= N)`, it's a constriant  validation.

Reply:
> `varchar(N)` is common, but we don't have a plan to support this datatype. So better throw errors to tell users that we don't support zero-filled fixed varchar datatype.

Yes, only let the parser pass is OK.
It works now in mysql handler:
```
mysql> create table ttt1(a varchar(10));

mysql> desc ttt1;
+-------+---------+------+---------+-------+
| Field | Type    | Null | Default | Extra |
+-------+---------+------+---------+-------+
| a     | VARCHAR | NO   |         |       |
+-------+---------+------+---------+-------+

```